src/NUnitFramework/framework/Internal/RuntimeFramework.cs

```diff
@@ -381,11 +381,18 @@ private static bool IsNetCore_Internal()
         }
 #endif
 
-        private static bool IsRuntimeTypeName(string name)
+        private static RuntimeType ParseRuntimeType(string s)
```

Comment:
This is the only new code. Runtime types are often passed in as strings and we allowed "NET" and "NetFramework". With the removal of  the `Net` enum value, this no longer worked.

I'll likely clean this up more when I work on the `PlatformAttribute`, but this will do for now.